client/src/track.ts

```diff
@@ -188,7 +188,10 @@ export default class Track extends BaseAnnotation {
    * Merge other into track at frame, preferring features from
    * self if there are conflicts
    */
-  merge(others: Track[]) {
+  merge(others: Track[], disableNotifier = false) {
+    if (disableNotifier) {
+      this.notifierEnabled = false;
+    }
```

Comment:
I'm ham fistedly abusing merge to get a track Object which has a combined begin/end so it can seek to the nearest frame of the track when selected using keyboard or the track list.  Maybe it would be best to refactor that out into something else and remove the `disableNotifier` stuff.